exercises/counter/description.md

```diff
@@ -1,3 +1,5 @@
+Design a test suite for a line/letter/character counter tool.
```

Comment:
Good spot. 

That is the location required so that the README will still be rendering identically.